README.md

```diff
@@ -1,43 +1,95 @@
-[![Ruff](https://img.shields.io/endpoint?url=https://raw.githubusercontent.com/astral-sh/ruff/main/assets/badge/v2.json)](https://github.com/astral-sh/ruff) ![PyPI - Python Version](https://img.shields.io/pypi/pyversions/nyaml) [![PyPI](https://img.shields.io/pypi/v/nyaml)](https://pypi.org/project/nyaml/)  [![Pytest](https://github.com/FAIRmat-NFDI/nyaml/actions/workflows/pytest.yaml/badge.svg)](https://github.com/FAIRmat-NFDI/nyaml/actions/workflows/pytest.yaml)
```

Comment:
We can keep these, no?

Reply:
I removed it because after the PR merged to the main branch the docs would be transferred to the mkdoscs format. I do not know how to fit the badges in mkdocs format. I will also keep you in that PR for mkdocs. If is there any way I will add them in mkdocs format.

Reply:
But we can keep them in the README and only transfer the relevant part to mkdocs, no? We don't want to delete the readme after transfering the main doc parts to mkdocs.

Reply:
Sorry, you are right. Though almost all the text will be moved to the mkdocs, still we can keep a simple README.md file.

Reply:
Done!